matrix-sdk-android/src/main/res/values/strings.xml

```diff
@@ -158,16 +158,14 @@
 
     <!-- The 2 parameters will be members' name -->
     <string name="room_displayname_two_members">%1$s and %2$s</string>
+    <string name="room_displayname_3_members">%1$s, %2$s and %3$s</string>
+    <string name="room_displayname_4_members">%1$s, %2$s, %3$s and %4$s</string>
+    <string name="room_displayname_four_and_more_members">%1$s, %2$s, %3$s and %4$d others</string>
```

Comment:
Actually it is rather `room_displayname_`**five**`_and_more_members`, also it has to be a plurals. See the legacy `room_displayname_three_and_more_members`, just below